plugin/pkg/auth/authenticator/token/bootstrap/bootstrap_test.go

```diff
@@ -247,3 +288,75 @@ func TestTokenAuthenticator(t *testing.T) {
 		}()
 	}
 }
+
+func TestGetGroups(t *testing.T) {
+	tests := []struct {
+		name         string
+		secret       *api.Secret
+		expectResult []string
+		expectError  bool
+	}{
+		{
+			name: "not set",
+			secret: &api.Secret{
+				ObjectMeta: metav1.ObjectMeta{Name: "test"},
+				Data:       map[string][]byte{},
+			},
+			expectResult: []string{bootstrapapi.BootstrapDefaultGroup},
```

Comment:
nit: I'm a fan of using hard-coded strings in unit tests sometimes. It seems like some of these could use hard-coding as we want things to be in a certain way. If some constant is changed (unexpectedly), the unit test will fail, and I'm in favor of failing in that case than letting things silently pass